packages/react-components/src/components/TextFieldWithMention/mentionTagUtils.ts

```diff
@@ -331,7 +331,7 @@ export const updateHTML = (props: UpdateHTMLProps): { updatedHTML: string; updat
 
   for (let i = 0; i < tags.length; i++) {
     const tag = tags[i];
-    if (tag.plainTextBeginIndex === undefined) {
+    if (tag?.plainTextBeginIndex === undefined) {
```

Comment:
Hi @vhuseinova-msft, could you help confirm whether we should continue when tag is undefined?

Reply:
Looks like the logic stayed the same. In this case, if `tag` is undefined, the next tag will be retrieved in the next loop iteration (`const tag = tags[i];`)